elements/head.html

```diff
@@ -0,0 +1,50 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        <!-- Actual Tag Name; Whatever the tag name is.  -->
+        &lt;tagname>
```

Comment:
yeah we were simply using the &lt to render the html <......I also am unfamiliar with parsing it